Makefile

```diff
@@ -48,3 +48,7 @@ build-image:
 
 .PHONY: install
 install: install-frontend install-backend
+
+.PHONY: example
+example: 
+	cd docs && oc apply -f prometheus-datasource-example.yaml && oc apply -f prometheus-dashboard-example.yaml
```

Comment:
```suggestion
	cd docs && oc apply -f prometheus-datasource-example.yaml && oc apply -f prometheus-dashboard-example.yaml

```

Reply:
Committed and squashed into the latest push.